utils/db.py

```
@@ -64,13 +64,12 @@ def get_hitter_info_by_id(player_id: int):
         return None
         
     try:
-        with connection:
-            with connection.cursor() as cursor:
+         with connection, connection.cursor() as cursor:
```

Pylint (3.3.7)

[warning] Bad indentation. Found 9 spaces, expected 8

(W0311)